x-pack/solutions/security/plugins/security_solution/common/api/endpoint/actions/response_actions/upload/upload.schema.yaml

```diff
@@ -39,8 +43,41 @@ components:
               properties:
                 overwrite:
                   type: boolean
+                  description: "Overwrite the file on the host if it already exists."
+                  example: false
                   default: false
             # File extends Blob - any binary data will be base-64 encoded
             file:
               type: string
+              description: "The file content to be uploaded or a relative path to the file. Note the path must be preceded with @."
+              example: "@fix-malware.sh"
```

Comment:
Not sure I understand this here... the `@` mentions here are very specific to using `curl` (maybe?) and I would say you probably want to keep this generic. The `file` param must be the file's binary content and must be sent using `Content-Type: multipart/form-data`

Reply:
You are right, I copied over an example from https://www.elastic.co/guide/en/security/8.17/upload-api.html
I went with more generic description here - https://github.com/elastic/kibana/pull/212510/commits/c1e844033141f5564f72e32ca0359c7afd223332